src/applications/static-pages/i18Select/createI18Select.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { Provider } from 'react-redux';
+
+export default function createI18Select(store, widgetType) {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+
+  const translatableLinks = new Set([
+    '/coronavirus-veteran-frequently-asked-questions/',
+    '/coronavirus-veteran-frequently-asked-questions-esp/',
+    '/coronavirus-veteran-frequently-asked-questions-tag/',
+    '/health-care/covid-19-vaccine/',
+    '/health-care/covid-19-vaccine-esp/',
+    '/health-care/covid-19-vaccine-tag/',
+  ]);
+  const isTranslatable = translatableLinks.has(document.location.pathname);
+  if (!isTranslatable) return;
+  const baseUrls = {
+    faq: {
+      en: '/coronavirus-veteran-frequently-asked-questions/',
+      es: '/coronavirus-veteran-frequently-asked-questions-esp/',
+      tag: '/coronavirus-veteran-frequently-asked-questions-tag/',
+    },
+    vaccine: {
+      en: '/health-care/covid-19-vaccine/',
+      es: '/health-care/covid-19-vaccine-esp/',
+      tag: '/health-care/covid-19-vaccine-tag/',
+    },
+  };
+  const isFaq = document.location.pathname.includes(
+    `/coronavirus-veteran-frequently-asked-questions`,
+  );
+
+  if (root) {
+    import('./I18Select').then(module => {
+      const I18Select = module.default;
+      ReactDOM.render(
+        <Provider store={store}>
+          <I18Select baseUrls={isFaq ? baseUrls.faq : baseUrls.vaccine} />
```

Comment:
@ncksllvn would love your thoughts on this approach. doesn't scale past two business use cases. perhaps just pass down the whole object?